sdk/rust/src/types.rs

```diff
@@ -12,9 +12,5 @@ typify_macro::import_types!(
 			type = "string",
 			format = "date-time"
 		} = jiff::Zoned,
-		{
-			type = "string",
-			format = "duration"
-		} = jiff::Span,
```

Comment:
Is there a reason to delete this in this fix, or is this a mistaken carry over change from some other work?

Reply:
This is related to removing the hc_github_token from the hc ready check.  Previously there were 3 elements in the 'ok' message tuple.  Now there are 2.  In the previous commit, this line was causing the Ubuntu test to fail.

Reply:
I'll check it out locally myself when I can. It doesn't make sense to me why this change to an environment variable would require removing the schema-generation mapping from the duration format to the `jiff::Span` type.

Reply:
Yeah this is certainly not correct, but I can't see what the failing ubuntu test was to diagnose what the proper fix is. Try restoring these lines?

Reply:
Agree with @j-lanson. I haven't been able to replicate the issue locally, but I'm on macOS. I don't see why this would fail on Ubuntu specifically though.

Reply:
No problem.  I'll put the lines back in and we can look at the ubuntu results.

Reply:
It looks like it passed Ubuntu. Whatever error there had been must have been because of something else. I'll approve and merge.